python/lsst/summit/utils/bestEffort.py

```diff
@@ -69,18 +68,14 @@ class BestEffortIsr():
     """
     _datasetName = 'quickLookExp'
 
-    def __init__(self, repodir, *,
+    def __init__(self, *,
                  extraCollections=[], defaultExtraIsrOptions={}, doRepairCosmics=True, doWrite=True):
-        if repodir not in ALLOWED_REPOS:
-            raise RuntimeError('Currently only NCSA and summit repos are supported')
         self.log = logging.getLogger(__name__)
 
-        location = _repoDirToLocation(repodir)
-        collections = (LATISS_SUPPLEMENTAL_COLLECTIONS[location] if location in
-                       LATISS_SUPPLEMENTAL_COLLECTIONS.keys() else []) + LATISS_DEFAULT_COLLECTIONS
+        collections = LATISS_DEFAULT_COLLECTIONS
         self.collections = extraCollections + collections
         self.log.info(f'Instantiating butler with collections={self.collections}')
-        self.butler = dafButler.Butler(repodir, collections=self.collections,
+        self.butler = dafButler.Butler('LATISS', collections=self.collections,
```

Comment:
Why not just use the `butlerUtils.makeDefaultLatissButler)` now that it exists?  Can't you pass the `run` to that method, since it already has the ability to make a writeable butler?